app/assets/javascripts/setupv2.js

```diff
@@ -1,6 +1,8 @@
 (function() {
   var POLL_INTERVAL = 1000;
-  var create_flash_container,
+  var PROGRESS_HALF_LIFE = 1000;
```

Comment:
🎨 You could probably make this smaller so that the animation looks even smoother

Reply:
The animation duration is the `PROGRESSION_ANIMATION_DURATION` constant. 
To clarify, which constant do you think I should change?
*  `PROGRESSION_ANIMATION_DURATION` 
*  `PROGRESS_HALF_LIFE`

Reply:
I'll give it a try!